### PR TITLE
feat: create onboarding

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Add a onboard Screen
Users can add their name

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 
 function App() {
-  return <div className="App"></div>;
+const[userName,setUserName] = useState("")
+const[route,setRoute] = useState("onboard")
+  return <div style = {{background: `url(${bg1}) center center/cover no-repeat`}} className="App" >
+     { route === "onboard" && <UserOnBoard userName ={userName} setUserName ={setUserName} setRoute={setRoute}/>}
+     { route === "main" && <Main/>}
+  </div>;
 }
```
Comment (harshitpaliwal95) [inline src/App.js:13]: good use of conditional radering

**Review by harshitpaliwal95 — APPROVED** [review]
code looks good
and good file folder structure

**Review by Sharmaryan — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+import { useState } from "react";
 
 function App() {
-  return <div className="App"></div>;
+const[userName,setUserName] = useState("")
+const[route,setRoute] = useState("onboard")
+  return <div style = {{background: `url(${bg1}) center center/cover no-repeat`}} className="App" >
+     { route === "onboard" && <UserOnBoard userName ={userName} setUserName ={setUserName} setRoute={setRoute}/>}
+     { route === "main" && <Main/>}
+  </div>;
```
Comment (Chiragmodi01) [inline src/App.js:12]: check `outlet` by router for condition rendering

File: src/App.js [hunk 3]
```diff
@@ -1,8 +1,15 @@
-import logo from "./logo.svg";
 import "./App.css";
+import { UserOnBoard,Main } from "./screens/index";
+import {bg1} from "./Assets/index"
+import { useState } from "react";
 
 function App() {
-  return <div className="App"></div>;
+const[userName,setUserName] = useState("")
+const[route,setRoute] = useState("onboard")
```
Comment (Chiragmodi01) [inline src/App.js:8]: I don't think router is needed for this extension.

File: src/screens/Onboard/UserOnBoard.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.onboard{
+    display:flex;
+    justify-content:center;
+    align-items:center;
```
Comment (Chiragmodi01) [inline src/screens/Onboard/UserOnBoard.css:4]: You can create a utility class for all these three flex properties, you'll need them many times in the ui

**Review by Chiragmodi01 — COMMENTED** [review]
Code is concise, added few sanity comments.
Looks good to merge.

**Pull request merged**